### PR TITLE
add atom/rss feed

### PR DESCRIPTION
**Pull request by kkkrist**
Would you consider adding back the rss/atom feed? Maybe something simple like this would do? Thanks :)

### CODE REVIEW

File: app/page.js [hunk 1]
```diff
@@ -25,6 +32,7 @@ export default async function Home() {
   posts.sort((a, b) => {
     return Date.parse(a.date) < Date.parse(b.date) ? 1 : -1;
   });
+  await generateFeed(metadata, posts);
```
Comment (gaearon) [inline app/page.js:35]: This feels kinda hacky. Can we use a [route handler](https://nextjs.org/docs/app/api-reference/file-conventions/route) instead? Does that work with static export?

**Review by gaearon — CHANGES_REQUESTED** [review]
thanks! would love help with this but want to tweak the implementation to be more idiomatic.

File: app/page.js (hunk 1 above)
Comment (gaearon) [reply app/page.js:35]: https://nextjs.org/docs/app/building-your-application/routing/route-handlers#non-ui-responses

Comment (kkkrist) [reply app/page.js]: Yes, you're right. I've updated the PR to use route handlers instead. It still seems to work fine with static exports.

**Comment by gaearon** [comment]
i tried running the atom feed through validator and it says author is missing. mind adding that? thanks!

File: app/atom.xml/route.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { getPosts, metadata } from "../page";
+import { generateFeed } from "../feed";
+
+export async function GET() {
+	const posts = await getPosts();
```
Comment (gaearon) [inline app/atom.xml/route.js:5]: nit: two spaces for indent

**Comment by gaearon** [comment]
thanks!!!

**Pull request merged**